src/test/java/sdu/sem2/se17/domain/persistenceinterface/ProductionHandlerTest.java

```diff
@@ -21,8 +22,8 @@
     @BeforeEach
     void setUp() {
         if (connectToDb){
-            dataSource = new DataSource("jdbc:postgresql://localhost:5432/", "postgres", "postgres");
-            handler = new ProductionHandlerImpl(dataSource, new CreditHandlerImpl(dataSource));
+            dataSource = new DataSource("jdbc:postgresql://localhost:5432/tv2", "postgres", "postgres");
```

Comment:
Database name is set in DataSource, method expects just the server url.
```suggestion
            dataSource = new DataSource("jdbc:postgresql://localhost:5432/", "postgres", "postgres");
```